src/CMakeLists.txt

```diff
@@ -115,7 +115,10 @@ endif (WIN32 OR CYGWIN)
 
 add_definitions (-DWITH_BROKER)
 
-set (MOSQ_LIBS ${MOSQ_LIBS} ${OPENSSL_LIBRARIES})
+set (MOSQ_LIBS ${MOSQ_LIBS})
+if(WITH_TLS)
```

Comment:
this seems like a standalone bug fix? ie, valid, but out of scope for a QNX support commit

Reply:
Ok, I'll create another PR for this